docs/doc/14-sql-commands/00-ddl/103-aggregating-index/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: AGGREGATING INDEX
+---
+import IndexOverviewList from '@site/src/components/IndexOverviewList';
+import EEFeature from '@site/src/components/EEFeature';
+
+<EEFeature featureName='AGGREGATING INDEX'/>
+
+### Why Aggregating Index?
+
+The aggregating index's primary function is to boost query performance, particularly in scenarios involving data filtering and selection. It achieves this by precomputing and storing query results separately in blocks, eliminating the need to scan the entire table and thereby speeding up data retrieval.
```

Comment:
The main scenarios are actually queries with aggregation (eg. MIN, MAX, SUM, ....).